eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/KzgRetriever.java

```diff
@@ -25,10 +25,11 @@ public class KzgRetriever {
   private static final Map<String, String> TRUSTED_SETUP_FILES_BY_NETWORK = Maps.newHashMap();
 
   public static KZG getKzgWithLoadedTrustedSetup(final Spec spec, final String network) {
-    if (!spec.isMilestoneSupported(SpecMilestone.DENEB)) {
-      return KZG.NOOP;
+    if (spec.isMilestoneSupported(SpecMilestone.DENEB)
+        || spec.isMilestoneSupported(SpecMilestone.ELECTRA)) {
+      return getKzgWithLoadedTrustedSetup(network);
     }
-    return getKzgWithLoadedTrustedSetup(network);
+    return KZG.NOOP;
```

Comment:
Could we have a task to make master pass reference tests fo FULU? I think we have several test executors already made, just needs revisiting

Reply:
I'll work on it.